plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/navigator/DBNResource.java

```diff
@@ -426,6 +429,9 @@ protected IStatus run(DBRProgressMonitor monitor) {
                                         true,
                                         monitor.getNestedMonitor());
                                 } else {
+                                    if (DBWorkbench.isDistributed() && !CommonUtils.equalObjects(otherResource.getProject(), resource.getProject())) {
```

Comment:
Projects are global, so their references must match. Is that correct?

```suggestion
                                    if (DBWorkbench.isDistributed() && otherResource.getProject() != resource.getProject()) {
```